### PR TITLE
Feature/hls bcov player

### PR DESCRIPTION
**Pull request by jordicenzano**
Needs to update player data & deploy

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+.*
+!.gitignore
```
Comment (philcluff) [inline .gitignore]: Add newline at the end of file.

File: deploy.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+aws s3 sync . s3://BUCKET/PATH --exclude ".*"
```
Comment (philcluff) [inline deploy.sh]: Should we add a "make public" action here too?

File: index.html [hunk 3]
```diff
@@ -33,7 +33,8 @@ <h3 class="text-muted">Phil's Players</h3>
         <li class="list-group-item"><a href="videojs-contrib-hls.html" >VideoJS with Contrib HLS</a></li>
         <li class="list-group-item"><a href="videojs-contrib-dash.html" >VideoJS with Contrib Dash (With Widevine)</a></li>
         <li class="list-group-item"><a href="dashjs.html" >DashJS (DashIF Reference Player)</a></li>
-        <li class="list-group-item"><a href="bc-player.html" >Brightcove Player (Dash CENC with Silverlight fallback)</a></li>
```
Comment (philcluff) [inline index.html:36]: Let's undo this rename so that we don't break people existing bookmarks.

File: bc-player-hls.html [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+          <label for="manifest">HLS Manifest:</label>
+          <input type="text" class="form-control" id="manifest" name="manifest">
+        </div>
+        <button type="submit" class="btn btn-primary btn-lg">Play!</button><br><br>
+        <button type="button" id="demo" class="btn btn-info btn-sm">Load Demo Manifest (from videoJS)</button>
+      </form>
+    </div>
+
+    <div class="jumbotron">
+      <video id="example" width="600" height="300" data-account="BCOV_ACCOUNT" data-player="BCOV_PLAYER_DATA" data-embed="default" class="video-js vjs-default-skin center-block" controls></video>
```
Comment (philcluff) [inline bc-player-hls.html]: TODO: Phil to find an OK player URL.

File: deploy.sh (hunk 2 above)
Comment (jordicenzano) [reply deploy.sh]: Depending how you are planning to manage the bucket properties...
But perhaps could be a good change

File: index.html (hunk 3 above)
Comment (jordicenzano) [reply index.html:36]: Done

File: .gitignore (hunk 1 above)
Comment (jordicenzano) [reply .gitignore]: Done

File: deploy.sh (hunk 2 above)
Comment (philcluff) [reply deploy.sh]: Dropping this file for now...

**Review by philcluff — APPROVED** [review]

**Comment by jordicenzano** [comment]
Nice! thanks

**Pull request merged**